src/main/scala/no/ndla/audioapi/AudioApiProperties.scala

```diff
@@ -40,7 +40,7 @@ object AudioApiProperties extends LazyLogging {
   val SearchServer = propOrElse("SEARCH_SERVER", "http://search-audio-api.ndla-local")
   val SearchRegion = propOrElse("SEARCH_REGION", "eu-central-1")
   val RunWithSignedSearchRequests = propOrElse("RUN_WITH_SIGNED_SEARCH_REQUESTS", "true").toBoolean
-  val SearchIndex = "audios"
+  lazy val SearchIndex = propOrElse("SEARCH_ALIAS", "audios")
```

Comment:
Utgangspunktet mitt er at vi har så mye løse deler fra før at jeg forsøker å holde nede antall nye :) 

Er det noen grunn til å tro at elasticsearch-instansen utenfor koden allerede har en index ved navn "audios"? (siden du legger inn mulighet for å overstyre navnet). Og siden du uansett "hardkoder" det i testklassen, så er det jo ikke "runtimestyrt" uansett...